File: internal/content/testdata/bench/go/maximize-div-plus-mod.go
func max_div_plus_mod(l int, r int, a int) int {
	best := r/a + r%a
	multiple := (r / a) * a
	candidate := multiple - 1
	if l <= candidate && candidate <= r {
		val := candidate/a + candidate%a
		if val > best {
			best = val
		}
	}
	return best
}

File: internal/content/testdata/bench/rust/maximize-div-plus-mod.rs
fn max_div_plus_mod(l: i64, r: i64, a: i64) -> i64 {
    let mut best = r / a + r % a;
    let multiple = (r / a) * a;
    let candidate = multiple - 1;
    if l <= candidate && candidate <= r {
        best = best.max(candidate / a + candidate % a);
    }
    best
}
